src/lambda/discordbot/lambda_handler.py

```diff
@@ -88,23 +30,32 @@ def lambda_handler(event, context):
     except Exception as e:
         raise Exception(f"[UNAUTHORIZED] Invalid request signature: {e}")
 
-    # check if message is a ping
-    body = event.get('body-json')
-    if ping_pong(body):
-        return PING_PONG
+    request = Request(event.get("body-json"))
+    response = ""
 
-    if app_command(body):
-        if blep(body):
-            # start_server()
-            return BLEP
-
-    # dummy return
-    return {
-            "type": RESPONSE_TYPES['MESSAGE_NO_SOURCE'],
-            "data": {
-                "tts": False,
-                "content": "BEEP BOOP",
-                "embeds": [],
-                "allowed_mentions": []
-            }
-        }
+    # check if message is a ping
+    if request.is_ping():
+        return GenericResponse.PONG_RESPONSE
+
+    if request.is_app_command():
+        if request.get_command() == Commands.BLEP:
+            response = ServerMenu(ResponseType.MESSAGE).run()
+
+    if request.is_component_interaction():
+        if request.user.get("id") != "105165905075396608":
+            response = MessageResponse(
```

Comment:
this should 100% be an embed or something not a new message to prevent spam 
![DiscordPTB_2021-09-19_23-29-35](https://user-images.githubusercontent.com/1351680/133963672-6f602997-506d-4177-8150-c6bbf10de167.png)

Reply:
Created an issue to track this, will do later https://github.com/Darklight-03/minecraft-cloudformation-server/issues/11